Makefile.core.mk

```diff
@@ -343,20 +343,26 @@ undeploy-olm: verify-kubeconfig operator-sdk ## Undeploy the operator from an ex
 	kubectl delete ztunnel.sailoperator.io --all --all-namespaces --wait=true
 	$(OPERATOR_SDK) cleanup $(OPERATOR_NAME) --delete-all -n ${NAMESPACE}
 
-.PHONY: deploy-example
-deploy-example: deploy-example-openshift ## Deploy an example Istio resource to an existing OCP cluster. Same as `deploy-example-openshift`.
+.PHONY: deploy-istio
+deploy-istio: verify-kubeconfig ## Deploy a sample Istio resource (without CNI) to an existing cluster.
+	kubectl create ns istio-system || echo "namespace istio-system already exists"
+	kubectl apply -f chart/samples/istio-sample-kubernetes.yaml
```

Comment:
Rename the file as well?

Reply:
I thought of renaming the file while submitting the PR, but was not sure if its used in other scripts. So did not make the change. But i agree, that we can remove one of them and rename it as `chart/samples/istio-sample.yaml` 

@FilipB do you think its okay to rename the file?
i.e., delete `chart/samples/istio-sample-kubernetes.yaml` and rename `chart/samples/istio-sample-openshift.yaml` to `chart/samples/istio-sample.yaml`
For some background, please see this - https://github.com/istio-ecosystem/sail-operator/issues/545#issue-2767638253

Reply:
To make it easy for review, I made the necessary changes in commit#2. PTAL.